README.md

```diff
@@ -8,16 +8,30 @@
 
 ---
 
-**ProbNum implements probabilistic numerical methods in Python.** Such methods solve numerical problems from linear
-algebra, optimization, quadrature and differential equations using _probabilistic inference_. This approach captures 
-uncertainty arising from _finite computational resources_ and _stochastic input_. 
+ProbNum is a Python toolkit for solving numerical problems in linear algebra, optimization, quadrature and
+differential equations. ProbNum solvers not only estimate the solution of the numerical problem, but also its
+uncertainty (the error) which certainly arises from finite computational resources, discretization, and stochastic input.
```

Comment:
just phrasing the uncertainty as error is misleading i think, since it ignores cross-correlation and also has the connotation of worst case error. This begs the question of why care about PN at all. 

"which ~certainly~" unnecessary and a duplication of uncertainty

Reply:
Hm, I was thinking about this, too when writing. The error is not identical with a standard deviation (standard error). It;s meant as "statistical error" or as some PN papers call it, the "discretization error". Both include the correlations. However, I wasn't happy with either since statistical error perhaps implies aleatoric uncertainty, and discretization error is only true for continuous domains (linear solvers are the exception there). Hence, I just said "error".

I find only writing "uncertainty" sub-optimal as it is a lesser understood (and less well defined term?) as e.g., statistical error, so I want to have a proper technical term in there. We could write "(the error of the solution)". This should be fairly agnostic. 

I removed the 'certainly'.